conda/core/subdir_data.py

```diff
@@ -553,6 +554,11 @@ def _get_base_url(self, repodata: dict, with_credentials: bool = True) -> str:
         return self.url_w_subdir
 
 
+@deprecated(
+    "24.9",
+    "25.3",
+    addendum="Use `conda.core.models.records.PackageRecord.make_feature_record` instead.",
+)
 def make_feature_record(feature_name):
     # necessary for the SAT solver to do the right thing with features
     pkg_name = f"{feature_name}@"
```

Comment:
Let's just return `PackageRecord.feature(feature_name)` here to not duplicate the implementation and only leave a stub, basically.

Reply:
Good catch! Done.